lib/pbench/server/api/resources/upload_api.py

```diff
@@ -127,9 +127,14 @@ def _put(self, args: ApiParams, request: Request, context: ApiContext) -> Respon
         identified rather than authorizing against an existing resource.
 
         Args:
-            filename: A filename matching the metadata of the uploaded tarball
-            access: The desired access policy (default is "private")
-            metadata: Metadata key/value pairs to set on dataset
+            params: API parameters
```

Comment:
s/params:/args:/ ?

(The function signature has `args` for the first argument/parameter.)

Reply:
True, but not really worth another commit?

Reply:
That was unclear, which is why I "approved"; and, under the present model, this open conversation doesn't block the merge....